### PR TITLE
Add animation to knob and value

### PR DESCRIPTION
**Pull request by kl3sk**

### CODE REVIEW

**Review by kramer99 — COMMENTED** [review]
I'll accept this if you make it disabled by default, and explain its use in the README, right after the value display function example.

**Comment by kl3sk** [comment]
Is this better for you ?

File: README.md [hunk 1]
```diff
@@ -90,4 +91,18 @@ textColor | String | Color of the value text | #000000
 strokeWidth | Number | Thickness of the arcs | 17
 valueDisplayFunction | Function | Custom function to alter the display text | `(v) => v`
 responsive | Boolean | Use `%` instead of `px` | false
-animation | Object | Animation config | {<br>animated: true,<br>animateValue: true,<br>animationDuration: 2000 (in ms),<br>animationFunction: 'ease-in-out'<br>}
+
+
+## Animation:
```
Comment (kramer99) [inline README.md]: Sorry, I should have been clearer in my last comment. Move this section up above the Properties table, and provide a HTML snippet of how to supply the animation settings.

File: README.md [hunk 2]
```diff
@@ -79,6 +79,7 @@ The only required property is `value`.
 Option | Type | Description | Default
 -------|------|-------------|--------
 value | Number | Use the `v-model` attribute to set the value of the control | none
+animation | Object | Animation config | {<br>animated: true,<br>animateValue: true,<br>animationDuration: 2000 (in ms),<br>animationFunction: 'ease-in-out'<br>}
```
Comment (kramer99) [inline README.md]: Move this table entry back to the bottom, also change what's in the default column.

Comment (kl3sk) [reply README.md]: Ok i’ll Move it back to the bottom. 

How do you want to write this section ?
It is an object with default, so entire object should be in the cell.

File: README.md (hunk 1 above)
Comment (kl3sk) [reply README.md]: Ok i’ll Provide a snippet as soon as possible.

File: README.md (hunk 2 above)
Comment (kramer99) [reply README.md]: I would move the example object into the description column, like this:

animation | Object | Optional animation config object: {<br>animated: true,<br>animateValue: true,<br>animationDuration: 2000 (in ms),<br>animationFunction: 'ease-in-out'<br>} | null

**Pull request merged**